### DISCUSSION TITLE
Proxy settings does not working since v1.189.0 in VSCode on Macos

### DISCUSSION THREAD

**Discussion by andamound** [community/community#123789] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

MacOS: Apple M3 Pro with 14.4.1 (23E224)

VSCode:

Version: 1.89.1 (Universal)
Commit: dc96b837cf6bb4af9cd736aa3af08cf8279f7685
Date: 2024-05-07T05:14:24.611Z (1 wk ago)
Electron: 28.2.8
ElectronBuildId: 27744544
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Darwin arm64 23.4.0

Copilot extension version: v1.191.0

Copilot does not use proxy settings for VSCode
Curl correct working with my proxy:

❯ curl -x http://myproxy:3128/ -i -L https://copilot-proxy.githubusercontent.com/_ping
HTTP/1.1 200 Connection established

HTTP/2 200 
content-security-policy: default-src 'none'; sandbox
content-type: application/json
strict-transport-security: max-age=31536000
content-length: 48
date: Tue, 14 May 2024 13:37:14 GMT
x-github-backend: Kubernetes
x-github-request-id: EC2A:7C6E0:145EDAA:15822F5:6643690A

{"now":1715693834,"status":"ok","ns1":"200 OK"}

But extension does not use proxy (i have an error while connecting to copilot).

This is only on MacOS for me. This is working fine on Windows with WSL.

Dependent issues: https://github.com/orgs/community/discussions/123651
https://github.com/orgs/community/discussions/123675

**Comment by devm33** [comment]
Hi @andamound, sorry for the trouble here, could you share how you are setting the proxy url? Is it in `http.proxy` in your `settings.json`?

**Reply by andamound** [reply]
Yes, "http.proxy": "http://login:password@ip_address:3128/" in my settings.json

change real login, pass and ip

**Reply by devm33** [reply, marked as answer]
Got it, thank you. Please update the Copilot extension to version 1.192.0 to resolve this issue. Sorry again for the inconvenience.
